styles.css

```diff
@@ -1,6 +1,108 @@
-body{
-    color: #bae1ff;
+:root {
```

Comment:
I like how you have used a :root selector because it compliments these colour elements as a local scope and helps when using the 'var' function to insert this colour schemes globally, this is a much precise way in using this scheme more commonly used throughout the site, well done 🙌